cln/cln_interceptor.go

```diff
@@ -164,7 +150,14 @@ func (i *ClnHtlcInterceptor) intercept() error {
 					interceptorClient.Send(i.defaultResolution(request))
 					i.doneWg.Done()
 				}
-				interceptResult := i.interceptor.Intercept(nextHop, paymentHash, request.Onion.ForwardMsat, request.Onion.OutgoingCltvValue, request.Htlc.CltvExpiry)
+
+				scid, err := basetypes.NewShortChannelIDFromString(request.Onion.ShortChannelId)
+				if err != nil {
+					interceptorClient.Send(i.defaultResolution(request))
+					i.doneWg.Done()
```

Comment:
shouldn't we return here?

Reply:
Yes, we should. That was already a bug above there.